tests/unit/test_handlers.py

```diff
@@ -501,33 +501,6 @@ def test_presigned_url_casing_changed_for_rds(self):
         )
         self.assertIn('X-Amz-Signature', params['PreSignedUrl'])
 
-    def test_500_status_code_set_for_200_response(self):
```

Comment:
If we are leaving the method there for backward compatibility, why are we deleting the tests?  (I see the names were reused below, but we should be making new tests for those)

Reply:
Got it, I will add them back.